packages/truffle-db/src/artifacts/schema.ts

```diff
@@ -84,32 +73,7 @@ export const schema = mergeSchemas({
           deployedSourceMap: sourceMap
         }) => ({ bytes, sourceMap })
       },
-      contractType: {
-        fragment: `... on ContractObject {
-          name: contractName
-        }`,
-        resolve: (obj, {}, context, info) => info.mergeInfo.delegateToSchema({
-          schema: jsonSchema,
-          operation: "query",
-          fieldName: "contract",
-          args: {
-            name: obj.name
-          },
-          context,
-          info,
-        })
-      }
-    },
-
-    ContractType: {
-      name: {
-        fragment: "... on ContractObject { name: contractName }"
-      },
-
-      createBytecode: {
-        fragment: "... on ContractObject { sourceMap, bytecode }",
-        resolve: ({ bytecode: bytes, sourceMap }) => ({ bytes, sourceMap })
-      }
+      
     },
   }
-});
+});
```

Comment:
You should turn on your editor settings to automatically add new lines at the end of files :)

Reply:
You know what, it totally does. I got a little overzealous when I was cleaning up other extraneous white spaces, haha.